apps/alarmeta/metadata.json

```diff
@@ -0,0 +1,15 @@
+{
+  "id": "alarmeta",
```

Comment:
my 2 cents: this is a widget so the name [_should_ start with **wid**](https://github.com/espruino/BangleApps#example-applications)

Reply:
done